src/components/Kanban.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+<div class="kanban-container">
+  <div class="kanban-board" v-for="category in categories" :id="category.name" :key="category.name" v-on:dragover="dragover" v-on:drop="drophandler">
+    {{category.name}}
+    <div v-for="(item,index) in boards[category.name]" draggable="true" v-on:dragstart="dragstart" :key="index">
+      {{item.name}}
+    </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import frappe from "frappejs";
+
+export default {
+  name: "Kanban",
+  props: {
+    msg: String
+  },
+  data: function() {
+    return {
+      boards: {
+        Open: [],
+        Closed: []
+      },
+      draggedItem: "",
+      categories: [{ name: "Open" }, { name: "Closed" }]
+    };
+  },
+  computed: {
+    meta() {
+      return frappe.getMeta("ToDo");
+    }
+  },
+  mounted() {
+    this.updateBoards();
+  },
+  methods: {
+    dragstart: function(e) {
+      e.dataTransfer.effectAllowed = "move";
+      const draggedItemName = String(e.target.innerHTML).trim();
+      let draggedItem;
+      var board;
+      for (board in this.boards) {
+        draggedItem = this.boards[board].find(
+          item => item.name === draggedItemName
+        );
+        if (typeof draggedItem !== "undefined") {
+          break;
+        }
+      }
+      this.categories.map(category => {
+        this.boards[category.name] = this.boards[category.name].filter(
+          item => item.name !== draggedItem.name
+        );
+      });
+      this.draggedItem = draggedItem;
+    },
+    drophandler: function(e) {
+      const dropDestination = e.target.id;
+      const draggedItem = this.draggedItem;
+      this.boards[dropDestination].push(draggedItem);
+      this.updateItem(draggedItem,dropDestination);
+    },
+    dragover: function(e) {
+      e.preventDefault();
+    },
+    async updateBoards() {
+      const open = await frappe.db.getAll({
+        doctype: "ToDo",
+        fields: ["name", ...this.meta.keywordFields, "status"],
+        filters: { status: "Open" }
+      });
+      const closed = await frappe.db.getAll({
+        doctype: "ToDo",
+        fields: ["name", ...this.meta.keywordFields, "status"],
+        filters: { status: "Closed" }
+      });
+      this.boards.Open = open;
+      this.boards.Closed = closed;
+    },
+    async updateItem(draggedItem,dropDestination) {
+      draggedItem = await frappe.getDoc('ToDo',draggedItem.name);
```

Comment:
you can access `draggedItem` here directly by using `this.draggedItem`

Reply:
Tried to do this but `update` function is not available on the `draggedItem` directly so I had to fetch the `doc`.